community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java

```diff
@@ -155,6 +183,37 @@ public final int get( TemporalField field )
         return temporal().get( field );
     }
 
+    public final AnyValue get( String fieldName )
```

Comment:
Would be good to have a javadoc comment explaining the purpose of this method for accessors.